configure.ac

```diff
@@ -140,6 +140,46 @@ AC_SUBST([COMMON_SSH_INCLUDE], '-I$(top_srcdir)/src/common-ssh')
 AC_SUBST([TERMINAL_LTLIB],   '$(top_builddir)/src/terminal/libguac_terminal.la')
 AC_SUBST([TERMINAL_INCLUDE], '-I$(top_srcdir)/src/terminal $(PANGO_CFLAGS) $(PANGOCAIRO_CFLAGS) $(COMMON_INCLUDE)')
 
+# pthread stack size
+AC_ARG_WITH(pthread_stack,
+            [AS_HELP_STRING([--with-pthread_stack=<size in bytes>],
+                            [explicitly set pthread stack size (8MB is recommended)])
```

Comment:
This may be confusing as "8MB" is not a value accepted by this option.

Reply:
Would you prefer it to read "8388608 (8MB)"

Reply:
That would probably be better (to avoid users trying things like `--with-pthread_stack=8MB`), though this may no longer be necessary based on your latest suggestion to simplify everything. I agree that it doesn't make sense to have a low-level option to tweak the stack size if it's known that the stack size absolutely must be at least 8MB.